qc-runner/src/dataIntegration.r

```diff
@@ -97,8 +97,11 @@ run_dataIntegration <- function(scdata, config){
     umap_min_distance <- 0.3
     umap_distance_metric <- "euclidean"
 
+    # temporary to make sure we don't run integration if unisample
+    nsamples <- length(unique(scdata$samples))
+    
     # Currently, we only support Seurat V3 pipeline for the multisample integration
-   if(method=="seuratv4"){
+    if(nsamples > 1 && method=="seuratv4"){
```

Comment:
There is an statement in data-ingest regarding checking the multisamples, but it is good to have a double check

https://github.com/biomage-ltd/data-ingest/blob/58be8a403d82439b8135bb180b71575247a64fa4/src/4_Prepare_experiment.r#L229